modin/tests/pandas/utils.py

```diff
@@ -914,6 +915,103 @@ class NoModinException(Exception):
     pass
 
 
+def eval_general_interop(
```

Comment:
Should we move this function to modin/tests/pandas/native_df_mode/utils.py? Can we re-use eval_general inside this function?

Reply:
I will move this to modin/tests/pandas/native_df_mode/utils.py, 

Regarding re-using eval_general, the problem is the main functionality of eval_general is to execute the function, the function executed in eval_general just needs to take in one dataframe, as  the second dataframe is derived from this single dataframe for binary operations,. In eval_general_interop however we would need to pass two dataframes to function being executed, which would need refactoring